picard/ui/mainwindow.py

```diff
@@ -729,7 +730,8 @@ def add_files(self):
         formats.sort()
         extensions.sort()
         formats.insert(0, _("All Supported Formats") + " (%s)" % " ".join(extensions))
-        files, _filter = QtWidgets.QFileDialog.getOpenFileNames(self, "", current_directory, ";;".join(formats))
+        #files, _filter = QtWidgets.QFileDialog.getOpenFileNames(self, "", current_directory, ";;".join(formats))
```

Comment:
This commented line needs to be removed in final patch.

Reply:
Yeah these lines will not be there in final patch.